### PR TITLE
Update f7xx HAL drivers and CMSIS

### PR DESCRIPTION
**Pull request by TDhaouST**
-     system(F7): update STM32F7xx CMSIS Drivers to v1.2.8
-     system(F7) update STM32F7xx HAL Drivers to v1.3.0

### CODE REVIEW

**Review by fpistm — APPROVED** [review]
LGTM

File: system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md [hunk 1]
```diff
@@ -5,13 +5,8 @@
   * STM32F2: 2.2.5
   * STM32F3: 2.3.6
   * STM32F4: 2.6.8
-<<<<<<< HEAD
   * STM32F7: 1.2.7
```
Comment (ABOSTM) [inline system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md:8]: New CMCIS version for F7 is 1.2.8

**Review by ABOSTM — COMMENTED** [review]
@TDhaouST,
can you please push a new PR with the correct F7 CMSIS version? Thanks

File: system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md (hunk 1 above)
Comment (fpistm) [reply system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md:8]: > New CMCIS version for F7 is 1.2.8

Nice catch @ABOSTM 
I will update it. I missed that when reviewed before the merge.
Looking deeply, it was weird as a merge conflict has been committed then remove the next commit. Really bad git usage method ended to an error in the file...

https://github.com/stm32duino/Arduino_Core_STM32/pull/1761/commits/7ea25271b0b516f85adb3c5c2e507a9ee343a641#diff-c2e123dabb25a1e1586c28bbe165ce41276ae6a37ac5445e5eca80300eb58297R8-R14

https://github.com/stm32duino/Arduino_Core_STM32/pull/1761/commits/91789d53bcee9c29a20a08bda9fcf9aa38a48ecc#diff-c2e123dabb25a1e1586c28bbe165ce41276ae6a37ac5445e5eca80300eb58297L8-L14

Comment (fpistm) [reply system/Drivers/CMSIS/Device/ST/STM32YYxx_CMSIS_version.md:8]: fixed with https://github.com/stm32duino/Arduino_Core_STM32/commit/90b3c4fe0e8d6f701a8ea324f923c9a34247d6db

**Comment by fpistm** [comment]
One more issue fixed:
https://github.com/stm32duino/Arduino_Core_STM32/commit/f0a043e4de30c9812e0256f51613f20c02c9f842

**Pull request merged**